app/models/course/condition/level.rb

```diff
@@ -1,4 +1,8 @@
 class Course::Condition::Level < ActiveRecord::Base
   acts_as_condition
   validates :minimum_level, numericality: { greater_than: 0 }
+
+  def title
+    I18n.t('activerecord.attributes.models.level_condition.title', value: minimum_level)
```

Comment:
I just realised you can't use the AR scope for attributes, because that's used for translating the column name to the humanised form.

So `activerecord.models` and `activerecord.attributes` are out.

How about `activerecord.models.attributes`, since we won't have a plural model named `attributes`? Then `activerecord.models.attributes.course.condition.level.title` should work?